src/Scenario/Resolver.php

```diff
@@ -40,11 +50,20 @@ public function resolve(array $scenarios): array
     {
         $instances = [];
 
+        $this->sorter->sortScenarios($scenarios);
+
         foreach ($scenarios as $step) {
-            $instance = $this->container->create(
-                $step['type'],
-                $this->resolveParams($step['arguments'] ?? [], $step['name'])
-            );
+            if (!$step['type']) {
+                $instance = $this->container->create(DummyStep::class, [
```

Comment:
Why do we have `DummyStep`?

Reply:
To add information into cloud.log about skipped tests.

Reply:
We can remove this Class but in this case, cloud.log won't contain information about skipped steps. I think this information is important for us in case of escalation.